operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -448,12 +448,12 @@ def validateNonDynamicClusterReferencedByAtLeastOneServer(self, cluster):
     for server in self.env.getDomain().getServers():
       if self.env.getClusterOrNone(server) is cluster:
         return
-    self.addError("The non-dynamic cluster " + self.name(cluster) + " is not referenced by any servers.")
+    self.addError("The non-dynamic WebLogic cluster " + self.name(cluster) + " is not referenced by any servers.")
```

Comment:
The WLS documentation does not use the term "non-dynamic" cluster (and I don't think that we should either), only dynamic or configured, https://docs.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/clust/dynamic_clusters.html#GUID-DA7F7FAD-49AA-4F3D-8A05-0D9921B96971.

Reply:
done